### PR TITLE
DIG-1240: API middleware service to handle queries across federated instances

### PR DESCRIPTION
**Pull request by OrdiNeu**
# Issues
[DIG-1240](https://candig.atlassian.net/browse/DIG-1240)

# Description
This is the submodule and docker-compose for the new query microservice.

# Linked issues
- [Data portal PR](https://github.com/CanDIG/candig-data-portal/pull/96)
- [Query PR](https://github.com/CanDIG/query/pull/1)
- [Federation PR](https://github.com/CanDIG/federation_service/pull/40)

# To test
- From this PR, that nothing is broken.
- This PR requires a change to your .env file

[DIG-1240]: https://candig.atlassian.net/browse/DIG-1240?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: etc/env/example.env [hunk 1]
```diff
@@ -248,6 +248,14 @@ TYK_INGEST_API_SLUG=ingest
 TYK_INGEST_API_TARGET=http://${CANDIG_DOMAIN}:${CANDIG_INGEST_PORT}
 TYK_INGEST_API_LISTEN_PATH=ingest
 
+## api - query
+QUERY_VERSION=1.0.1
```
Comment (daisieh) [inline etc/env/example.env]: We usually put the stuff that is not Tyk-related in its own section...

**Review by daisieh — APPROVED** [review]
One tiny comment about order in .env, but otherwise LGTM.

**Comment by OrdiNeu** [comment]
(force push to get up to date with `develop`. Will wait for the integration tests, but they ran fine on my machine)

File: etc/env/example.env (hunk 1 above)
Comment (OrdiNeu) [reply etc/env/example.env]: Fixed in [eb56e52](https://github.com/CanDIG/CanDIGv2/pull/291/commits/eb56e523a3c1ee8c0c0d7098f366863cf6453825) :heavy_check_mark:

**Pull request merged**